src/components/List/ListHeader/_list-header.scss

```diff
@@ -0,0 +1,44 @@
+.list-header {
+  display: flex;
+  align-items: center;
+  background: $ui-01;
+  padding-left: $spacing-04;
+  border-bottom: 1px solid $ui-03;
+  height: rem(56px);
+  &--title {
+    @include type-style('productive-heading-02');
+    flex: 1;
+  }
+
+  &--btn-container {
+    display: flex;
+    padding: $spacing-03;
+    align-items: center;
+    & > * {
+      margin-left: $spacing-03;
+    }
+  }
+
+  &--search {
+    padding: 0 1rem;
```

Comment:
should use spacing token here for this rem value

Reply:
1 rem = $spacing-05